nautobot/docs/additional-features/jobs.md

```diff
@@ -35,7 +35,7 @@ In any case, each module holds one or more Jobs (Python classes), each of which
 For example, we can create a module named `devices.py` to hold all of our jobs which pertain to devices in Nautobot. Within that module, we might define several jobs. Each job is defined as a Python class inheriting from `extras.jobs.Job`, which provides the base functionality needed to accept user input and log activity.
 
 +/- 2.0.0
-    All job classes must now be registered with `nautobot.core.celery.register_jobs` on module import. For plugins providing jobs, the module containing the jobs must be imported by the plugin's `__init__.py` and the `register_jobs` method called at import time. The `register_jobs` method accepts one or more job classes as arguments.
```

Comment:
Is the note about loading the job module in the app's `__init__.py` unnecessary because the job module always gets loaded by [the plugin ready callback](https://github.com/nautobot/nautobot/blob/8746766e141cd5d5c84d260f5d629d331c12da06/nautobot/extras/plugins/__init__.py#L144C2-L144C2)?